lib/matrix.dart

```diff
@@ -25,7 +25,6 @@ export 'fake_matrix_api.dart' show FakeMatrixApi;
 
 export 'src/client.dart';
 export 'src/database/database_api.dart';
-export 'src/database/hive_database.dart';
```

Comment:
Since we're removing a DB option, shouldn't this be a breaking change even though it was deprecated beforehand?

Reply:
Mentioned it in the commit message